---a/net/minecraft/command/server/CommandPardonPlayer.java
+++b/net/minecraft/command/server/CommandPardonPlayer.java
@@ -1,8 +1,6 @@
 package net.minecraft.command.server;
 
-import com.mojang.authlib.GameProfile;
 import java.util.List;
+
 import net.minecraft.command.CommandBase;
 import net.minecraft.command.CommandException;
 import net.minecraft.command.ICommandSender;
@@ -9,56 +9,48 @@
 import net.minecraft.server.MinecraftServer;
 import net.minecraft.util.BlockPos;
 
-public class CommandPardonPlayer extends CommandBase
-{
+import com.mojang.authlib.GameProfile;
+
+public class CommandPardonPlayer extends CommandBase {
+
     private static final String __OBFID = "CL_00000747";
 
-    public String getName()
-    {
+    public String getName() {
         return "pardon";
     }
 
-    public int getRequiredPermissionLevel()
-    {
+    public int getRequiredPermissionLevel() {
         return 3;
     }
 
-    public String getCommandUsage(ICommandSender sender)
-    {
+    public String getCommandUsage(ICommandSender sender) {
         return "commands.unban.usage";
     }
 
-    public boolean canCommandSenderUse(ICommandSender sender)
-    {
+    public boolean canCommandSenderUse(ICommandSender sender) {
         return MinecraftServer.getServer().getConfigurationManager().getBannedPlayers().isLanServer() && super.canCommandSenderUse(sender);
     }
 
-    public void execute(ICommandSender sender, String[] args) throws CommandException
-    {
-        if (args.length == 1 && args[0].length() > 0)
-        {
+    public void execute(ICommandSender sender, String[] args) throws CommandException {
+        if (args.length == 1 && args[0].length() > 0) {
             MinecraftServer minecraftserver = MinecraftServer.getServer();
             GameProfile gameprofile = minecraftserver.getConfigurationManager().getBannedPlayers().isUsernameBanned(args[0]);
 
-            if (gameprofile == null)
-            {
-                throw new CommandException("commands.unban.failed", new Object[] {args[0]});
-            }
-            else
-            {
+            if (gameprofile == null) {
+                throw new CommandException("commands.unban.failed", new Object[]{args[0]});
+            } else {
                 minecraftserver.getConfigurationManager().getBannedPlayers().removeEntry(gameprofile);
-                notifyOperators(sender, this, "commands.unban.success", new Object[] {args[0]});
+                notifyOperators(sender, this, "commands.unban.success", new Object[]{args[0]});
             }
-        }
-        else
-        {
+        } else {
             throw new WrongUsageException("commands.unban.usage", new Object[0]);
         }
     }
 
-    public List addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos)
-    {
+    public List addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos) {
         return args.length == 1 ? getListOfStringsMatchingLastWord(args, MinecraftServer.getServer().getConfigurationManager().getBannedPlayers().getKeys()) : null;
     }
-}
+}
